docs/phase-2-migration/workflow-of-content.md

```diff
@@ -0,0 +1,65 @@
+## Workflow of content
+
+The diagram below shows the workflow of content within the Specialist Publisher
+app. Each arrow corresponds to an action that transitions content from one state
+to another.
+
+![workflow](./workflow.png)
+
+## Explanation
+
+Content can either be in a draft, published or unpublished state. Draft content
+is accessible via the draft stack (draft-origin) which is used by writers and
+editors who would like to preview their content before it goes live. Unpublished
+content shows a 410 "This content is no longer here" page on GOV.UK.
+
+Content must be created in a draft state. After that, there are three choices.
+The content can either be edited, an attachment could be added or the content
+could be published. If an attachment is added, its metadata is added to the
+content item in the details hash.
+
+If the content is published, a new draft can be created from it. Alternatively,
+it can transition to unpublished. When content is unpublished, a new draft must
+be created before it can be published again. Similarly, if the content is
+published and there is also a draft, the draft must be published in order to
+unpublish the content.
+
+This workflow would be easier for our users to manage if we added the ability to
+discard a draft. This is something the platform supports but isn't implemented
+by this application.
+
+## API requests
+
+This section lists the API requests made at each transition. In total,
+Specialist Publisher speaks to four external services.
+
+**new / edit**
+
+- Publishing API: PUT /v2/content
+- Publishing API: PATCH /v2/links
```

Comment:
Ahh, I understand your point. I agree that this should probably happen at publish time. That being said, we always send the same links in Specialist Publisher Rebuild, so it doesn't actually matter in this case. I'll add a note to that documentation to that effect.